### ISSUE TITLE
Datasets in solutions not useable

### ISSUE THREAD

**Issue by Ecofoot** [Oysiyl/DataCamp_Solutions_Python#1]
Oysiyl,
I have downloaded the zip files to your solutions and have unzipped them. I am having difficulty being able to access and view the Naive Bee (both projects-predict and image loading) datasets; the bee_1.jpg (etc.) files and the "images.zip" file of bee photos for machine learning. 

It seems like these files may be corrupt.

How can I obtain this data to use off the DataCamp site such is in my own Anaconda python program etc.  I would like to view/analyze the data more on my own.

Thanks.

Rich 
"Ecofoot"

**Comment by dmitriy-kisil** [comment]
Hmm, interesting.. After completing each project you can download notebook withprovided data for him via DataCamp. I checked  - and zip are really corrupt. It's look like problem on DataCamp side. Or DataCamp decided to disallow access to datasets from users.

**Comment by Ecofoot** [comment]
For the Deep Learning solution for Naive Bees part 9:

Why do you use "model.evaluate(..." on holdout set rather than 
pretrained_cnn.evaluate(.....  as on test set?
Thanks.

-Rich
*
**# load pre-trained model**
**pretrained_cnn = keras.models.load_model('datasets/pretrained_model.h5')**
**
**_# evaluate model on test set_**
**score =**/pretrained_cnn.evaluate/**(x_test, y_test, verbose=0)**
**print('Test loss:', score[0])**
**print('Test accuracy:', score[1])**
**
**print("")**
**
**_# evaluate model on holdout set_**
**eval_score = **/model.evaluate/**(x_eval, y_eval, verbose=0)*

**Comment by dmitriy-kisil** [comment]
Oh, I download an old version of the notebook! My fault( For now I updated this project. Thanks for the comments
